src/components/plots/ClassifierKneePlot.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { Vega } from 'react-vega';
+
+import EmptyPlot from './helpers/EmptyPlot';
+import generateSpec from '../../utils/plotSpecs/generateClassifierKneePlot';
+
+const ClassifierKneePlot = (props) => {
+  const {
+    config, expConfig, plotData, actions,
+  } = props;
+
+  const [plotSpec, setPlotSpec] = useState(config);
+
+  useEffect(() => {
+    if (config && plotData) {
+      setPlotSpec(generateSpec(config, expConfig, plotData));
+    }
+  }, [config, expConfig, plotData]);
+
+  const render = () => {
+    if (!plotData.length) {
+      return (
+        <EmptyPlot mini={config.miniPlot} />
+      );
+    }
+
+    return (
+      <center>
+        <Vega spec={plotSpec} renderer='canvas' actions={actions} />
+      </center>
+    );
+  };
+
+  return (
+    <>
```

Comment:
A bit of a nitpick - you can just do 
`    if (!plotData.length) {
      return (
        <EmptyPlot mini={config.miniPlot} />
      );
    }

    return (
      <center>
        <Vega spec={plotSpec} renderer='canvas' actions={actions} />
      </center>
    );` straight away instead of going into a return and calling the `render` function

Reply:
I don't think it is a nitpick, good point, I'll change this